cpp/lib/src/util.cc

```diff
@@ -64,6 +89,13 @@ Logger::Logger()
 }
 
 
+void Logger::redirectOutput(const int new_fd) {
```

Comment:
In the current version in ruschein branch, redirectOutput is directly implemented in util.h which is not part of this pull request. Did you forget to commit it?

Reply:
I did indeed! :-(